elemental/client.py

```diff
@@ -167,3 +169,17 @@ def get_input_devices(self):
         devices_info = sorted(
             devices_info, key=lambda d: int(d["id"]))
         return [dict(d) for d in devices_info]
+
+    def generate_preview(self, device_id):
+        url = f'{self.server_ip}/devices/{device_id}/preview'
+        headers = self.generate_headers(url)
+
+        # generate body
+        body = file_fixture('device_preview.xml')
```

Comment:
I think it's strange to see "fixture" being used outside testing context
https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming

maybe `read_template()`?